osu.Game.Modes.Osu/Objects/SliderCurve.cs

```diff
@@ -172,6 +172,9 @@ private Vector2 interpolateVertices(int i, double d)
         /// <param name="p1">End progress. Ranges from 0 (beginning of the slider) to 1 (end of the slider).</param>
         public void GetPathToProgress(List<Vector2> path, double p0, double p1)
         {
+            if (calculatedPath.Count == 0 && ControlPoints.Count > 0)
+                Calculate();

```

Comment:
yep, i get that.

but i think we need better logic for this kinda thing (cached.invalidate) anyway.

also this is still not run at gameplay as PositionAt is used very early in ctors